crates/cli/tests/reference/wasm-export-types.d.ts

```diff
@@ -2,17 +2,99 @@
 /* eslint-disable */
 export function example(a: number, b: bigint, c: any, d: string): string;
 export function example_128(a: bigint): bigint | undefined;
+/**
+ * Handler for `console.log` invocations.
+ *
+ * If a test is currently running it takes the `args` array and stringifies
+ * it and appends it to the current output of the test. Otherwise it passes
+ * the arguments to the original `console.log` function, psased as
+ * `original`.
+ */
+export function __wbgtest_console_log(args: Array<any>): void;
+/**
+ * Handler for `console.debug` invocations. See above.
+ */
+export function __wbgtest_console_debug(args: Array<any>): void;
+/**
+ * Handler for `console.info` invocations. See above.
+ */
+export function __wbgtest_console_info(args: Array<any>): void;
+/**
+ * Handler for `console.warn` invocations. See above.
+ */
+export function __wbgtest_console_warn(args: Array<any>): void;
+/**
+ * Handler for `console.error` invocations. See above.
+ */
+export function __wbgtest_console_error(args: Array<any>): void;
+export function __wbgtest_cov_dump(): Uint8Array | undefined;
+/**
+ * Runtime test harness support instantiated in JS.
+ *
+ * The node.js entry script instantiates a `Context` here which is used to
+ * drive test execution.
+ */
+export class WasmBindgenTestContext {
+  free(): void;
+  /**
+   * Creates a new context ready to run tests.
+   *
+   * A `Context` is the main structure through which test execution is
+   * coordinated, and this will collect output and results for all executed
+   * tests.
+   */
+  constructor();
+  /**
+   * Handle `--include-ignored` flag.
+   */
+  include_ignored(include_ignored: boolean): void;
+  /**
+   * Handle filter argument.
+   */
+  filtered_count(filtered: number): void;
+  /**
+   * Executes a list of tests, returning a promise representing their
+   * eventual completion.
+   *
+   * This is the main entry point for executing tests. All the tests passed
+   * in are the JS `Function` object that was plucked off the
+   * `WebAssembly.Instance` exports list.
+   *
+   * The promise returned resolves to either `true` if all tests passed or
+   * `false` if at least one test failed.
+   */
+  run(tests: any[]): Promise<any>;
+}
 
 export type InitInput = RequestInfo | URL | Response | BufferSource | WebAssembly.Module;
 
 export interface InitOutput {
   readonly memory: WebAssembly.Memory;
   readonly example: (a: number, b: bigint, c: any, d: number, e: number) => [number, number];
   readonly example_128: (a: bigint, b: bigint) => [number, bigint, bigint];
-  readonly __wbindgen_export_0: WebAssembly.Table;
+  readonly "__wbgt__reference_test::example_test": (a: number) => void;
+  readonly __wbgtest_cov_dump: () => [number, number];
```

Comment:
The fix is visible here :) `__wbgt__reference_test::example_test` is quoted now. 🎉